internal/usecase/repo/asset.go

```diff
@@ -11,7 +11,7 @@ import (
 type Asset interface {
 	Save(context.Context, *asset.Asset) error
 	Remove(context.Context, id.AssetID) error
-	FindByTeam(context.Context, id.TeamID, *usecase.Pagination) ([]*asset.Asset, *usecase.PageInfo, error)
+	FindByTeam(context.Context, id.TeamID, *string, *string, *usecase.Pagination) ([]*asset.Asset, *usecase.PageInfo, error)
```

Comment:
It is very confusing so using a struct is good:

```go
// defined at internal/usecase/repo/asset.go
type AssetFilter struct {
	Sort       *asset.SortType
	Keyword    *string
	Pagination *usecase.Pagination
}
```

```suggestion
	FindByTeam(context.Context, id.TeamID, AssetFilter) ([]*asset.Asset, *usecase.PageInfo, error)
```